### PR TITLE
*dab

### PR DESCRIPTION
**Pull request by actioninja**
## About The Pull Request

Adds dabbing

## Why It's Good For The Game

immersive dabbing roleplay SOUL balance community vote oranges bad coder dont play jkdsfjkaejrklsdjfksnweltknatkldtrjawettkl

## Changelog
:cl:
add: *dab
/:cl:

### CODE REVIEW

**Comment by Ordonis** [comment]
c u r s e d

**Review by Ordonis — CHANGES_REQUESTED** [review]
Yes hello I would like to suggest the removal of the additions to the emote file. All the other changes are good. Reason: Dab.

**Comment by ATH1909** [comment]
YES.

I'd suggest removing the chance to get a special brain trauma, though- people already have other people intentionally fail lobotomies so they can get a shot at getting an imaginary friend, bluespace prophet, godwoken...

Actually, keep it in- dabbing so hard/well that you receive a guardian dabgel that only you can see/become woke and see the holes that you tore in spacetime with your dab/become possessed by the god of dabbing sounds rad as hell.

**Comment by actioninja** [comment]
bruh sound effect #2

**Comment by PKPenguin321** [comment]
monky

File: code/modules/mob/living/carbon/emote.dm [hunk 1]
```diff
@@ -15,6 +15,74 @@
...
+			if(3)
+				selected_part = BODY_ZONE_R_LEG
+			if(4)
+				selected_part = BODY_ZONE_R_ARM
+			if(5)
+				selected_part = BODY_ZONE_L_ARM
+		if(selected_part == "SEIZURE_TIME")
+			C.visible_message("<span class='warning'>[C] starts seizing and convulsing!</span>", "<span class='danger'>As you hit the dab, your whole body begins spasming!</span>")
+			var/datum/brain_trauma/T
+			if(prob(80))
```
Comment (Ziiro) [inline code/modules/mob/living/carbon/emote.dm]: `(prob(80))`

n i c e

**Comment by ATH1909** [comment]
IT WORKS

AHAHAHAHAHAHAHAAAAAAA

https://cdn.discordapp.com/attachments/326826573951008769/570886097375068170/200_dabs.PNG

I HAVE DABBED OVER 200 TIMES IN A SINGLE ROUND

AND LIVED

AHAHAHAHAHAHAHAAAAAAAAAAAAAAA

**Comment by ATH1909** [comment]
https://www.reddit.com/r/SS13/comments/bh654h/local_lizardperson_dabs_over_200_times_and/

**Comment by gy1ta23** [comment]
Plagiarism is not okay

**Comment by ATH1909** [comment]
> Plagiarism is not okay

Plagiarism?! Those screenshots are mine, not someone else's. You can ask Plapatin himself about it, and he'll tell you that I play Hits-The-Ball (in addition to Jack Jackson and a few other characters).

**Comment by ATH1909** [comment]
... Wait, do you need arms to dab?

**Comment by Ziiro** [comment]
>... Wait, do you need arms to dab?

Apparently not. I got myself nuggeted by too much dabbing and I could still do it.

**Comment by adamkad1** [comment]
anyone who dabs should be valid

**Comment by Ziiro** [comment]
Some actual issues:

1) If you dab while stunned/unconscious, the emote doesn't play but it still runs through all the bad shit
2) If you dab your legs off, they aren't actually removed? It's really weird. Examine says  they're missing, but there the sprite is still there and you can keep wearing shoes
3) I have seen at least 25% of a shift dab itself to death every shift

Dabbing your way to superpowers is legit. Gamble big, win big prizes.

**Comment by ZeWaka** [comment]
necro bruh

**Pull request closed without merging**